test/integration/scheduled-action-integration.ts

```diff
@@ -0,0 +1,346 @@
+
+      await testSpace.updateScheduledAction({
+        scheduledActionId: sys.id,
+        version: sys.version,
+        payload: {
+          ...payload,
+          scheduledFor: {
+            datetime: updatedSchedule,
+            timezone: 'Europe/Kiev',
+          },
+        },
+      })
+
+      const actions = await testSpace.getScheduledActions({
+        'environment.sys.id': payload.environment.sys.id,
+        'sys.status': 'scheduled',
+        'sys.id[in]': scheduledAction.sys.id,
+      })
+      const updatedAction = actions.items[0]
+
+      expect(updatedAction.entity).to.deep.equal(makeLink('Asset', asset.sys.id))
+      expect(updatedAction.action).to.eql('unpublish')
+      expect(updatedAction.scheduledFor).to.deep.equal({
+        datetime: updatedSchedule,
+        timezone: 'Europe/Kiev',
+      })
+
+      // cleanup
+      await updatedAction.delete()
+    })
+
+    test('fails to update unsupported property of scheduled actions', async () => {
+      const updatedSchedule = new Date(new Date(datetime).getTime() + ONE_DAY_MS).toISOString()
+
+      const scheduledAction = await testSpace.createScheduledAction({
+        entity: makeLink('Asset', asset.sys.id),
+        environment: makeLink('Environment', environment.sys.id),
+        action: 'unpublish',
+        scheduledFor: {
+          datetime,
+        },
+      })
+
+      expect(scheduledAction.entity).to.eql(makeLink('Asset', asset.sys.id))
+      expect(scheduledAction.scheduledFor).to.deep.equal({
+        datetime,
+      })
+
+      const { sys, ...payload } = scheduledAction
+
+      try {
+        await testSpace.updateScheduledAction({
+          scheduledActionId: sys.id,
+          version: sys.version,
+          payload: {
+            entity: makeLink('Entry', TestDefaults.entry.testEntryId),
+            environment: makeLink('Environment', environment.sys.id),
+            action: 'publish',
+            scheduledFor: {
+              datetime: updatedSchedule,
+            },
+          },
+        })
+      } catch (error) {
+        const parsedError = JSON.parse(error.message)
+        expect(parsedError.status).to.eql(400)
+        expect(parsedError.statusText).to.eql('Bad Request'),
+          expect(parsedError.message).to.eql(
+            'Can only update scheduleFor.datetime and scheduleFor.timezone properties'
+          )
+      }
+
+      // cleanup
+      await scheduledAction.delete()
+    })
+
+    test('delete scheduled action', async () => {
+      const scheduledAction = await testSpace.createScheduledAction({
+        entity: makeLink('Asset', asset.sys.id),
+        environment: makeLink('Environment', environment.sys.id),
+        action: 'unpublish',
+        scheduledFor: {
+          datetime,
+        },
+      })
+
+      // Deletes a Release and all its Release Actions
+      await scheduledAction.delete()
+
+      const response = await testSpace.getScheduledActions({
```

Comment:
wondering if wouldn't be better to try to find the specific scheduled action.

What I am thinking is that if for some reason another test is running this might become flakey since you querying for all scheduled actions.

Reply:
wait a minute, we actually don't have a normal get added to the SDK? 🤦🏽

Reply:
yeah, I wanted to, but then I heard "please not in the same PR", so I cut the ticket instead